met-api/src/met_api/models/map.py

```diff
@@ -0,0 +1,39 @@
+"""WidgetMap model class.
+
+Manages the map widget
+"""
+from __future__ import annotations
+
+from sqlalchemy.sql.schema import ForeignKey
+
+from .base_model import BaseModel
+from .db import db
+
+
+class WidgetMap(BaseModel):  # pylint: disable=too-few-public-methods
```

Comment:
please also change the file name map.py to something like widget_map. I'm thinking map is a very general term that could apply to any potential map component that isn't related to widgets

Reply:
done